CONTRIBUTING.md

```diff
@@ -0,0 +1,248 @@
+# Contributing
```

Comment:
file could be moved .github folder

Reply:
I think it should stay in the root: https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors